src/state_stack.cpp

```diff
@@ -20,7 +20,7 @@ State_Stack::State_Stack(STATES arg_init_state)
     const auto cwd = std::filesystem::current_path();
 
     auto modulepath = cwd;
-    modulepath.append("resources").append("modules");
+    modulepath.append("resources/modules"); // TODO: DLA MNIE MUSI KONIECZNIE BYC \\ ZAMIAST / WIEC ZROBCIE TO ZALEZNE OD PLATFORMY XD

```

Comment:
ALE WINDOWS I TAK RESPEKTUJE /

Reply:
Myslisz, ze gdybym nie mial przez to problemu, to bym to zmienial?

Reply:
A dobra wiem co sie stalo XD nvm

Reply:
cóż, ogólnie to niby powinno się to jeszcze inaczej zrobić, za pomocą `std::filesystem::path / std::filesystem::path`, acz append powinien też być ok